packages/near-fast-auth-signer/src/lib/useAuthState.ts

```diff
@@ -63,8 +71,31 @@ export const useAuthState = (skipGetKeys = false): AuthState => {
             setAuthenticated(true);
           }
         }).catch(() => setAuthenticated(false));
+    } else {
+      checkFirestoreReady().then(async (isReady) => {
+        if (isReady) {
+          // @ts-ignore
+          const oidcToken = firebaseAuth.currentUser.accessToken;
+          if (window.fastAuthController.getLocalStoreKey(`oidc_keypair_${oidcToken}`)) {
+            const recoveryPK = await window.fastAuthController.getUserCredential(oidcToken);
+            const accountIds = await fetch(`${network.fastAuth.authHelperUrl}/publicKey/${recoveryPK}/accounts`)
```

Comment:
On this scenario accountIds will be type any, no? 

Can we defined the API call on a separated file, handle the error and properly type it's return? 

I'm placing the API calls on this file: https://github.com/near/fast-auth-signer/blob/main/packages/near-fast-auth-signer/src/api/index.ts

But we can also discus how we can organize it. 

I just don't like the idea of spreading API calls across the codebase and not typing it's return type.

Reply:
Agreed, let's throw an issue on the board to collect all of these and organize into service files wdyt? Unfortunately we are only now in a position to address tech debt

Reply:
Yeah I understand and agree 100%, let's start creating tickets for the tech debt when we face them.